client/reader/start/main.jsx

```diff
@@ -60,9 +60,9 @@ const Start = React.createClass( {
 			<Main className="reader-start">
 				<QueryReaderStartRecommendations />
 				<header className="reader-start__intro">
-					<h1 className="reader-start__title">{ this.translate( 'Welcome to the Reader' ) }</h1>
-					<p className="reader-start__description">{ this.translate( "Discover great stories and read your favorite sites' posts all in one place. Every time there are new updates to the sites you follow, you'll be the first to know!" ) }
-				</p>
+					<h1 className="reader-start__title">{ this.translate( 'Welcome to the WordPress.com Reader' ) }</h1>
+					<p className="reader-start__description">{ this.translate( "Discover great stories and read your favorite sites' posts all in one place. Every time there are new updates to the sites you follow, you'll be the first to know!" ) }</p>
+					<p className="reader-start__description">{ this.translate( "We've curated a list of content for you. Follow one or more sites to get started." ) }</p>
```

Comment:
I think "we've curated a list of content" could sound a bit dry.

How about "we've suggested some sites that you might enjoy" or similar?